lib/Core/Site/Values/Content.php

```diff
@@ -408,11 +421,17 @@ private function getInnerOwnerUser()
             return $this->innerOwnerUser;
         }
 
-        try {
-            $this->innerOwnerUser = $this->userService->loadUser($this->getContentInfo()->ownerId);
-        } catch (NotFoundException $e) {
-            // Do nothing
-        }
+        $userService = $this->userService;
```

Comment:
There's a difference in allowing anonymous access to user content and user object. While former should be allowed access for anonymous users, this here should not, since it contains sensitive data: username, hashed password and email address, so this needs to revert to the original code.